src/App.tsx

```diff
@@ -1,14 +1,47 @@
-function App() {
+import { useState } from 'react'
+import { Entity } from './types'
+import { LOREM_IPSUM } from './constants'
+
+const INITIAL_ENTITIES: Entity[] = [
+  {
+    title: 'John Smith',
+    subtitle: LOREM_IPSUM,
+  },
+  {
+    title: 'Elizabeth Doe',
+    subtitle: LOREM_IPSUM,
+  },
+]
+
+const App = () => {
+  const [entities, setEntities] = useState<Entity[]>(INITIAL_ENTITIES)
+
   return (
-    <div className="flex flex-col bg-anti-flash-white h-dvh">
+    <div className="flex flex-col bg-silver h-dvh">
       <header>
-        <h1 className="text-center font-bold text-lg py-4 uppercase">
+        <h1 className="text-center font-bold text-3xl py-4 uppercase">
           Your contacts
         </h1>
       </header>
       <main className="p-2 flex justify-center">
-        <div className="flex justify-center items-center bg-silver shadow-lg grow-1 w-full max-w-2xl text-white min-h-52 rounded">
-          list of contacts
+        <div className="flex flex-col items-center bg-anti-flash-white shadow-lg grow-1 w-full max-w-2xl min-h-52 rounded-lg p-4 gap-4">
+          {entities.map((entity) => {
+            const initialLetter = entity.title[0]
```

Comment:
Maybe default to something else if `entity.title[0]` is undefined? Like a question mark symbol??